example/app/config.yml

```diff
@@ -192,6 +192,13 @@ aimee-test:
   stackNameNoDash: aimeeTest
 
 
+  communityConfiguration:
+    ApiTokenDefault:
```

Comment:
I think you also need to change ApiTokenV1 too? Did you consider just having an auth flag and setting it to Google and setting the ApiToken endpoints somewhere internally? (I had made a comment about this before)

Reply:
@scisco what uses `ApiTokenV1`?

I believe the dashboard points to the `/token/` endpoint which goes to the `ApiTokenDefault` endpoint. If we had an interface using the `/v1/token` endpoint we could test it but otherwise I don't think it is necessary.

I'm not sure I understand how to implement the auth flag suggestion (Sorry if I missed your comment earlier). Do you mean to have a conditional on what modules is exported as `token` in @cumulus/api? E.g.:

```javascript
exports.token = if (process.env.OAUTH_PROVIDER === 'google') {
  require('./endpoints/googleToken');
} else {
  require('./endpoints/token');
};
```

This seems like it would work well - I guess the difference is:
1) With the environment variable, you can easily switch between auth providers (not sure if this is a good or bad thing)
2) Configuration of the environment variable would need be defined in lambdas.yml so we need to add additional `envs:` variable packages/api/config/api_default.yml and api_v1.yml
3) We could remove the changes to `packages/deployment/app/cumulus_api.template.yml`

@laurenfrederick is this what you meant? @scisco what do you think?

Reply:
I like the env variable approach better. As for how to configure it we can do something like this:

```yaml
ApiTokenDefault:
  handler: index.token
  timeout: 20
  memory: 512
  source: 'node_modules/@cumulus/api/dist/'
  apiRole: true
  urs: true
  envs:
    OAUTH_PROVIDER: '{{parent.oauth.provider}}'
    EARTHDATA_BASE_URL: '{{parent.urs_url}}'
```

and then add this to the default config:
```yaml
default:
  oauth:
    provider: earthdata
```